packages/parany/parany.7.0.0/opam

```diff
@@ -7,7 +7,7 @@ bug-reports: "https://github.com/UnixJunkie/parany/issues"
 dev-repo: "git+https://github.com/UnixJunkie/parany.git"
 depends: [
   "ocaml" {>= "4.03.0"}
-  "dune"
+  "dune" {>= "1.6.0"}
```

Comment:
```suggestion
  "dune" {< "2.0"}
```
Oh, looking at the error in #15398, I think this requires a upper-bound constraint instead
```
#=== ERROR while compiling parany.7.0.0 =======================================#
# context              2.0.5 | linux/x86_64 | ocaml-base-compiler.4.09.0 | git+file:///home/opam/opam-repository
# path                 ~/.opam/4.09/.opam-switch/build/parany.7.0.0
# command              ~/.opam/4.09/bin/dune build -p parany -j 2
# exit-code            1
# env-file             ~/.opam/log/parany-25-42ef32.env
# output-file          ~/.opam/log/parany-25-42ef32.out
### output ###
# Info: Creating file dune-project with this contents:
# | (lang dune 2.0)
# | (name parany)
# File "src/dune", line 15, characters 2-36:
# 15 |   (action
# 16 |   (run ./test.exe 16 800)))
# Error: 'action' was deleted in version 2.0 of the dune language. Use a rule
# stanza with the alias field instead

- Info: Creating file dune-project with this contents:
- | (lang dune 2.0)
- | (name parany)
- File "src/dune", line 15, characters 2-36:
- 15 |   (action
- 16 |   (run ./test.exe 16 800)))
- Error: 'action' was deleted in version 2.0 of the dune language. Use a rule
- stanza with the alias field instead
```